src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -18,6 +18,7 @@ public UnixDomainSocketTest(ITestOutputHelper output)
             _log = TestLogging.GetInstance();
         }
 
+        [ActiveIssue(27542)]
         [OuterLoop] // TODO: Issue #11345
```

Comment:
This issue has been closed, update the comment? https://github.com/dotnet/corefx/issues/11345
Do we need to point it to this one now? https://github.com/dotnet/corefx/issues/14519

cc @davidsh

Reply:
> 11345

There are tons of references to this still around the repo.  They should be cleaned up, but I'm not going to do that in this change.

Reply:
The test should still be left as outerloop.